Python/View_and_Combine_STL/README.md

```diff
@@ -0,0 +1,24 @@
+# View and Combine STL
+
+STL files are an important part of 3D asset creation and management. STL files provide us with basic maps of 3D objects created/edited in applications like Blender/Unreal/Unity
+
+STL are mainly used to create 3D maps of environments like cityscapes and heightmaps of terrain. 
+
+## Running the Script
+* Create a virtual env
+* install numpy-stl, numpy and matplotlib or use requirements.txt
```

Comment:
Refer to the proper command: `Install the Packages by running the command: pip install -r requirements.txt`

Reply:
Resolved